songPlayed.php

```diff
@@ -0,0 +1,46 @@
+<?php
+require 'vendor/autoload.php';
+
+use Musicplayer\Database\SongDao;
+
+header("Access-Control-Allow-Origin: http://localhost:3000");
+
```

Comment:
consistency - spaces between all headers here but isn't in the others. Also more headers in this than in the others?